### PR TITLE
Dev arkanoid cris

### PR DESCRIPTION
**Pull request by Cotzv**
Creating the Arkanoid Template

### CODE REVIEW

**Review by jchoque87 — APPROVED** [review]

**Review by Milena-Rodriguez-Montecinos — APPROVED** [review]

**Review by stellith — APPROVED** [review]

**Review by virichino — APPROVED** [review]

File: config.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Config {
+    constructor(type,width,height){
```
Comment (jchoque87) [inline config.js:2]: the attributes are not use

File: index.html [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script src="https://cdn.jsdelivr.net/npm/phaser@3.15.1/dist/phaser-arcade-physics.min.js"></script>
+    <script src="main.js"></script>
```
Comment (jchoque87) [inline index.html:5]: To Responsive:
<meta name="viewport" content="width=device-width, initial-scale=1.0">

File: main.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        width: 800,
+        height: 600,
+        physics: {
+            default: 'arcade',
+            arcade: {
+                gravity: { y: 200 }
+            }
+        },
+        scene: {
+            preload: cargar,
```
Comment (J0c4) [inline main.js:12]: Please use english for all code

File: index.html (hunk 2 above)
Comment (jchoque87) [reply index.html:5]: Tag meta with name="viewport" content="width=device-width, initial-scale=1.0

**Pull request closed without merging**